revolv/project/views.py

```diff
@@ -21,11 +21,11 @@ class DonationLevelFormSetMixin(object):
     the Create Project and Update Project page.
     """
 
-    def get_donation_level_formset(self, extra = 2):
+    def get_donation_level_formset(self, extra=2):
```

Comment:
Weird, I didn't actually change that...must have been cought by one of the python flakers. I guess we should leave it in, it is PEP8 after all